backend/capellacollab/projects/capellamodels/routes.py

```diff
@@ -23,57 +22,57 @@
     ResponseModel,
     ToolDetails,
 )
+from .modelsources.git.routes import router as router_sources_git
+from .modelsources.t4c.routes import router as router_sources_t4c
+
+router = APIRouter(
+    dependencies=[
+        Depends(ProjectRoleVerification(required_role=ProjectUserRole.USER))
+    ],
+)
 
-router = APIRouter()
+
+router.include_router(
+    router_sources_git,
+    prefix="/{model_slug}/git",
+    tags=["Projects - Models - Git"],
+)
+router.include_router(
+    router_sources_t4c,
+    prefix="/{model_slug}/t4c",
+    tags=["Projects - Models - T4C"],
+)
 
 
-# FIXME: Add verification by dependency injection
-@router.get("/", response_model=t.List[ResponseModel])
-def list_in_project(
-    project=Depends(get_existing_project),
+@router.get("/", response_model=list[ResponseModel])
+def get_models(
+    project: DatabaseProject = Depends(get_existing_project),
     db: Session = Depends(get_db),
-    token=Depends(JWTBearer()),
-) -> t.List[ResponseModel]:
-    verify_project_role(project.name, token, db)
-    return [
-        ResponseModel.from_orm(model)
-        for model in crud.get_all_models_in_project(db, project.slug)
-    ]
+) -> list[DatabaseCapellaModel]:
+    return crud.get_all_models_in_project(db, project)
```

Comment:
You could actually return `project.models`, isn’t it?

Reply:
This should already be done here.